### PR TITLE
[Fix] Configuration of fetching in RSA Netwitness

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/21059

## Description
Fixed an issue with ***fetch-incidents*** when setting a *Fetch Limit*, it would drop older incidents if the fetched incidents number was greater than the limit.

Added new argument to *pageNumber* to ***netwitness-get-incidents*** command. The new argument allows the user to get incidents from a specific page, intended to be used with limit argument.

## Does it break backward compatibility?
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py [hunk 1]
```diff
@@ -296,17 +299,62 @@ def get_all_incidents(since=None, until=None, limit=None):
...
+        LOG('Requesting for page {}'.format(page_number))
+        response_body = get_incidents_request(
+            since=since,
+            until=until,
+            page_number=page_number
+        )
+        incidents = response_body.get('items')
+        # clear incidents after ignore_id
+        for inc in incidents:
+            if inc.get('id') == ignore_id:
```
Comment (yuvalbenshalom) [inline Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py]: lt's change ignore_id to last_fetched_id

File: Integrations/RSANetWitness_v11_1/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+- Fixed an issue with ***fetch-incidents*** when setting a *Fetch Limit*, it would drop older incidents if the fetched incidents number was greater than the limit.
```
Comment (amshamah419) [inline Integrations/RSANetWitness_v11_1/CHANGELOG.md]:
```suggestion
- Fixed an issue with ***fetch-incidents*** where setting a *Fetch Limit* would drop older incidents if the number of the fetched incidents was greater than the limit.
```

File: Integrations/RSANetWitness_v11_1/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
+- Added new argument to *pageNumber* to ***netwitness-get-incidents*** command. The new argument allows the user to get incidents from a specific page, intended to be used with limit argument.
```
Comment (amshamah419) [inline Integrations/RSANetWitness_v11_1/CHANGELOG.md]:
```suggestion
- Added the new argument *pageNumber* to  the ***netwitness-get-incidents*** command. The new argument allows the user to get incidents from a specific page and is intended to be used with limit argument.
```

**Comment by amshamah419** [comment]
Commit the changes and you’re good to go.

**Pull request merged**